keps/sig-network/2091-admin-network-policy/README.md

```diff
@@ -541,15 +542,14 @@ type AdminNetworkPolicyPort struct {
   // If present, only traffic on the specified protocol AND port will be matched.
   // +optional
   Port       *intstr.IntOrString
-    
+
   // If set, indicates that the range of ports from port to endPort, inclusive,
   // should be allowed by the policy. This field cannot be defined if the port field
   // is not defined or if the port field is defined as a named (string) port.
   // The endPort must be equal or greater than port.
-  // This feature is in Beta state and is enabled by default.
-  // It can be disabled using the Feature Gate "NetworkPolicyEndPort".
+  // This feature can be disabled using the Feature Gate "NetworkPolicyEndPort".
```

Comment:
No it can't :) We actually need to remove this since it will be built into the API from day 1

Reply:
Done